### DISCUSSION TITLE
Video Autoscaling

### DISCUSSION THREAD

**Discussion by LukeIGS** [kedacore/keda#2494] [Ideas]
Two issues in question for Selenium:
https://github.com/SeleniumHQ/selenium/issues/10018
https://github.com/SeleniumHQ/selenium/issues/9845

Keda may be able to help here it seems, per @tomkerkhove 

General gist is that Selenium Grid has certain requirements for video recording, around the ffmpeg container terminating correctly, and actually standing up in the first place based on capabilities sent to the driver.  Quick readup shows the Keda uses the selenium graphql layer to pull metrics to drive its events. so this should actually be possible since the video capability would be visible there.

**Comment by tomkerkhove** [comment]
Hi, I'm not sure I get the scenario as we simply autoscale workloads - Either by using ScaledObject (deamon) or ScaledJob (job, ~instance per message for example)

**Reply by LukeIGS** [reply]
So effectively, what it looks like is going on from keda's standpoint is that it would scale a separate job like chrome-video or firefox-video if i'm understanding the tests in place correctly which still falls back to needing a way send a proper termination message to the ffmpeg container; which feels a little out of scope for what keda seems to be to me.

**Reply by tomkerkhove** [reply]
Yes, that is correct. Nothing will sned a termination request with a job though. So it's up to the container to complete once it's done.
